src/System.Security.Claims/src/System/Security/Claims/ClaimTypes.cs

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//------------------------------------------------------------------------------
+
+using System.Runtime.InteropServices;
+
+namespace System.Security.Claims
+{
+    /// <summary>
+    /// Defines the claim types that are supported by the framework.
+    /// </summary>   
+    [ComVisible(false)]
```

Comment:
Correct @venkat-raman251 please go ahead and strip all the ComVisible attributes in these source files.